server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -644,8 +644,8 @@ public TableManager getTableManager() {
     return context.getTableManager();
   }
 
-  public AccumuloClient getClient() throws AccumuloSecurityException, AccumuloException {
```

Comment:
The master has a getContext method.  So this method could be dropped entirely.  May be able to change the impl of getClient to call getContext and then inline the method.

Reply:
Done in c491aff